custom_components/localtuya/common.py

```diff
@@ -201,6 +247,10 @@ async def close(self):
             await self._interface.close()
         if self._disconnect_task is not None:
             self._disconnect_task()
+        self.debug(
```

Comment:
Is this at the right level? IE, can self._interface and self._disconnect_task be None; yet this still logs that it disconnected; even if it did nothing?

Reply:
Well, I just want to make sure that that method is called and ends, regardless of whether the device was actually connected at that time, so I'd keep it like this.